sys/include/net/gnrc/icmpv6/error.h

```diff
@@ -76,6 +79,101 @@ gnrc_pktsnip_t *gnrc_icmpv6_error_time_exc_build(uint8_t code, gnrc_pktsnip_t *o
 gnrc_pktsnip_t *gnrc_icmpv6_error_param_prob_build(uint8_t code, void *ptr,
                                                    gnrc_pktsnip_t *orig_pkt);
 
+/**
+ * @brief   Sends an ICMPv6 destination unreachable message for sending.
+ *
+ * @param[in] code      The code for the message @see net/icmpv6.h.
+ * @param[in] orig_pkt  The invoking packet.
+ */
+static inline void gnrc_icmpv6_error_dst_unr_send(uint8_t code, gnrc_pktsnip_t *orig_pkt)
+{
+#ifdef MODULE_GNRC_IPV6
```

Comment:
Shouldn't his be rather a `ifdef MODULE_GNRC_ICMPV6_ERROR` or something like that?